Contents/Code/siteDigitalPlayground.py

```diff
@@ -41,7 +41,10 @@ def update(metadata,siteID,movieGenres,movieActors):
     try:
         sceneType = detailsPageElements.xpath('//div[contains(@class, "i01da7-0 bRuuus")]/a[2]')[0].text_content().strip()
     except:
-        sceneType = detailsPageElements.xpath('//div[contains(@class, "i01da7-0 bRuuus")]/a[1]')[0].text_content().strip()
+        try:
+            sceneType = detailsPageElements.xpath('//div[contains(@class, "i01da7-0 bRuuus")]/a[1]')[0].text_content().strip()
+        except:
+            sceneType = 'Scene'
```

Comment:
Could you add a line like:
`Log("SceneType check failed, assuming SceneType is Scene")`